src/types/Venue.ts

```diff
@@ -87,6 +71,35 @@ export interface Venue {
   showSecretPasswordForm?: boolean;
 }
 
+export interface VenueConfig {
+  theme: {
+    primaryColor: string;
+    backgroundColor?: string;
+  };
+
+  jazzbarConfig: VenueJazzbarConfig;
+  landingPageConfig: VenueLandingPageConfig; // @debt should this be potentially undefined, or is it guaranteed to exist everywhere?
+
+  memberEmails?: string[];
+  showRangers?: boolean;
+}
+
+export interface VenueJazzbarConfig {
+  tables?: Table[];
+}
```

Comment:
```suggestion
```

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/793/commits/d38a1bcc34f80038394d3705cf3e917120abc1f4